Source/WebCore/css/parser/CSSParserImpl.cpp

```diff
@@ -592,6 +602,138 @@ RefPtr<StyleRuleFontFace> CSSParserImpl::consumeFontFaceRule(CSSParserTokenRange
     return StyleRuleFontFace::create(createStyleProperties(m_parsedProperties, m_context.mode));
 }
 
+static std::pair<FontFeatureValuesType, unsigned> fontFeatureValuesTypeMappings(CSSAtRuleID id)
+{
+    switch (id) {
+    // Styleset has no maximum in theory, but in practise we don't consider more than 99 values.
+    case CSSAtRuleStyleset: return { FontFeatureValuesType::Styleset, 99 }; 
```

Comment:
Should there be a fixme with a link to a new bug to avoid having to use an arbitrary number here?

Reply:
I don't think we should try to support it because it will not be used by the platform.

I've added a link to the MDN sentence : "Note that values higher than 99 are valid, but don't map to any OpenType values and are ignored." https://developer.mozilla.org/en-US/docs/Web/CSS/@font-feature-values#syntax